tests/Detector.Tests/StaticSiteGeneratorHelperTest.cs

```diff
@@ -0,0 +1,106 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using Microsoft.Oryx.Detector.Hugo;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+
+namespace Microsoft.Oryx.Detector.Tests
+{
+    public class StaticSiteGeneratorHelperTest : IClassFixture<TestTempDirTestFixture>
```

Comment:
Could you also add HugoDetectorTest

Reply:
Since HugoDetector is just passing on the responsibility to `StaticSiteGeneratorHelper`, I added test for this type...I ideally want to remove this `StaticSiteGeneratorHelper` and keep all logic in HugoDetector itself.